invoker/core/src/main/java/com/google/cloud/functions/invoker/BackgroundFunctionExecutor.java

```diff
@@ -72,18 +72,20 @@ private BackgroundFunctionExecutor(FunctionExecutor<?> functionExecutor) {
 
     /** Returns the {@link FunctionKind} that the given class implements, if any. */
     static Optional<FunctionKind> forClass(Class<?> functionClass) {
-      return VALUES.stream().filter(v -> v.functionClass.isAssignableFrom(functionClass)).findFirst();
+      return VALUES.stream()
```

Comment:
I don't think this change is correct. The column limit for the Google style guide is 100 characters. It seems like the config used here is set to 80 or some other style.

https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

Reply:
In my experience, Java is pretty flexible about how it breaks up lines. @grant see [section 4.5](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) of the guide you linked: 

> **Note:** While the typical reason for line-wrapping is to avoid overflowing the column limit, even code that would in fact fit within the column limit may be line-wrapped at the author's discretion.

Reply:
Yes, there's some flexibility, but I don't think the PR formatting changes were justified for fixing severity.